adr/ADR-32.md

```diff
@@ -25,15 +25,15 @@ Service configuration relies on the following:
   `A-Z, a-z, 0-9, dash, underscore`.
 - `version` - a SemVer string - impl should validate that this is SemVer
 - `description` - a human-readable description about the service (optional)
-- `apiUrl`: (optional) - url pointing to user-defined API specification
-- `metadata` - (optional) an object of strings holding free form metadata about 
-   the deployed instance implemented consistently with [Metadata for Stream and Consumer ADR-33](ADR-33.md).
+- `metadata` - (optional) an object of strings holding free form metadata about
+  the deployed instance implemented consistently with
+  [Metadata for Stream and Consumer ADR-33](ADR-33.md).
 - `statsHandler` - an optional function that returns unknown data that can be
   serialized as JSON. The handler will be provided the endpoint for which it is
   building a `EndpointStats`
-- `endpoint` - an optional base endpoint configuration, consisting of valid NATS subject,
-  handler and optional request/response schema. The `name` of the base endpoint 
-  should be set to `default` by the client libraries.
+- `endpoint` - an optional base endpoint configuration, consisting of valid NATS
+  subject, handler and optional request/response schema. The `name` of the base
```

Comment:
still mentions schema here

Reply:
the word `schema` is about the `schema` for the PING response fields - let me remove that